ITEventsTests/FullEventPresenterTests/FullEventPresenterTests.swift

```diff
@@ -0,0 +1,188 @@
+import XCTest
+@testable import Promises
+@testable import ITEvents
+
+class FullEventPresenterTests: XCTestCase {
+    var presenter: IFullEventPresenter!
+    var view: FullEventViewMock!
```

Comment:
Could we please always postfix all the variables with "mock".
That way the only real implementations won't be postfixed:
```swift
var presenter: IFullEventPresenter! // real impl
var viewMock: FullEventViewMock! // mock impl
```

These hints are extremely important for code readability for example here:

```swift
presenter = FullEventPresenter(view: view, // mock?
                               selectedEventService: selectedEventService,  // mock?
                               dateFormatterService: DateFormatterServiceMock(),
                               similarEventsService: SimilarEventsStorageMock())
```
How do I know those variables are mocks? I need a hint, compare to this:
```swift
presenter = FullEventPresenter(view: viewMock, // now I see it's mock
                               selectedEventService: selectedEventServiceMock,  // now I see it's mock
                               dateFormatterService: DateFormatterServiceMock(),
                               similarEventsService: SimilarEventsStorageMock())
```

Could you please apply this for all the mock variables?